client/app/services/query.js

```diff
@@ -483,6 +507,7 @@ function QueryResource(
     }
 
     const parameters = this.getParameters();
+    parameters.applyPendingValues();
```

Comment:
This is the line that allows "Execute" button and kb shortcut to apply before running the query.